src/assets/a11y-viewer.css

```diff
@@ -222,8 +222,7 @@ main {
 }
 
 main.is-disabled {
-    pointer-events: none;
```

Comment:
The :before layer already blocks the clicks.
I tested this, and the `pointer-events: none` doesnt seem to have any value anymore.